add copilot seat analysis supports (#32)
github-copilot-resources/copilot-metrics-viewer: README.md

4. **Total Active Copilot Chat Users:** a bar chart that illustrates the total number of users who have actively interacted with Copilot over the past 28 days.

## Seat Analysis 

> ![image](https://github.com/DevOps-zhuang/copilot-metrics-viewer/assets/54096296/d1fa9d1d-4fab-4e87-84ba-7be189dd4dd0)

Reviewer: We might need to update this image once the changes have been implemented 🙏🏼

Reviewer: Issue created: [README Seat Analysis image update](https://github.com/github-copilot-resources/copilot-metrics-viewer/issues/35)

> 1. **Total Assigned:** This metric represents the total number of copilot assigned seats within current organization.

Reviewer: ```suggestion
1. **Total Assigned:** This metric represents the total number of Copilot seats assigned within current organization.
```

> 2. **Assigned But Never Used:** This metric shows the assigned but never used seats within current organization. the assigned time are also dispalyed in the below chart, recommend to check that date.

Reviewer: ```suggestion
2. **Assigned But Never Used:** This metric shows seats that were assigned but never within the current organization. The assigned timestamp is also displayed in the below chart.
```

> 3. **No Activity in Last 7 days:** never used seats. or seats used, but no activity in recent 7 days.

Reviewer: ```suggestion
3. **No Activity in the Last 7 days:** never used seats or seats used, but with no activity in the past 7 days.
```

> 4. **No Activity in last 7 days (Inluding Never used seats):** A table to display seats that have had no activity in the past 7 days, ordered by the date of last activity. Seats that were used earlier are displayed more prominently at the front.

Reviewer: ```suggestion
4. **No Activity in the last 7 days (including never used seats):**a table to display seats that have had no activity in the past 7 days, ordered by the date of last activity. Seats that were used earlier are displayed at the top.
```
